### PR TITLE
Uefi systemd img and osuosl images

### PR DESCRIPTION
**Pull request by jonesmz**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -40,7 +35,7 @@ pipeline
 		}
 		environment
 		{
-			PROJECT="GenPi64${INIT_SYSTEM}"
```
Comment (jonesmz) [inline Jenkinsfile:43]: I'm not sure that we want to merge the jenkinsfile changes.

would this be better done by adding the osuosl image as a third type?

File: config/patches/sys-apps/systemd/systemd-bootctl.patch
Comment (samip5) [inline config/patches/sys-apps/systemd/systemd-bootctl.patch:1]: Why is this needed?

**Review by samip5 — COMMENTED** [review]
The commit message doesn't make sense as I don't see anything particularly supporting X86_64 UEFI systemd?

**Comment by jonesmz** [comment]
> The commit message (on commit [2be9fd1](https://github.com/GenPi64/Build.Dist/commit/2be9fd13e5b2845901590b104017397abc4c196f)) doesn't make sense as I don't see anything particularly supporting X86_64 UEFI systemd?

This file is the config for an image that will only boot with Uefi.

[config/pyconfig/GenPi64UefiSystemdConfig.py](https://github.com/GenPi64/Build.Dist/pull/213/commits/2be9fd13e5b2845901590b104017397abc4c196f#diff-c4ddf750c189a9e16430522fa9f7ef99fa082989dc2460e1ec27ccc2a6b00af1)

it uses Uefi (systemd-boot) + a UnifiedKernelImage (Dracut + kernel).

Then the aarch64 version is based on the x86_64 version.

The x86_64 built image works on my local virt-manager(qemu) hypervisor, i'm still waiting for an aarch64 build to try.

File: config/patches/sys-apps/systemd/systemd-bootctl.patch
Comment (jonesmz) [reply config/patches/sys-apps/systemd/systemd-bootctl.patch:1]: Without the patch, the `bootctl` program that installs the Uefi stub loader into /efi/efi/whateversubdir will bail out because it detects that /efi/ inside of the chroot is not the root of the filesystem, whiich is a stupid check to require.

The patch is used to build systemd (for systemd-boot), then bootctl is run, then systemd is re-built without the patch.

I also added systemd to the list of programs to not install via binpkg, because gentoo's binpkg's dont track whether or not patches were applied during the compilation of the package, so if it picks up the binpkg, it might or might not have the patch.

Comment (jonesmz) [reply config/patches/sys-apps/systemd/systemd-bootctl.patch:1]: I reported the issue, and the patch, to systemd upstream. Unlikely anything will be done about it.

**Review by samip5 — APPROVED** [review]
